assets/scripts/free-trial-wc-payments.js

```diff
@@ -68,7 +68,8 @@
 			);
 
 			if ( stepItems.length === 3 ) {
-				const p = stepItems.querySelector( 'p' );
+				const { 2: setupCompleteStepElement } = stepItems;
+				const p = setupCompleteStepElement.querySelector( 'p' );
```

Comment:
Just curious can we directly access the node element?

```suggestion
				const p = stepItems[ 2 ].querySelector( 'p' );
```

Reply:
Could be possible with [nth-of-type](https://caniuse.com/?search=nth-of-type), but I'm not too picky at this moment

Reply:
that would work too! But I like to name things so that its clear what "element 2" is 😬